src/main/java/gregtech/api/recipes/crafttweaker/CTRecipe.java

```diff
@@ -22,6 +22,8 @@
 @SuppressWarnings("unused")
 public class CTRecipe {
 
+    // TODO YEET
```

Comment:
Were we going to remove CT compat? I don't remember that being the case.

Reply:
CT is causing so many issues and shitty code. I would like its compate to be gone. Only when proper conversion tools are available of course.